### ISSUE TITLE
groupedEach functionality

### ISSUE THREAD

**Issue by robsilva** [helpers/handlebars-helpers#248]
Hi there,

What a terrific library! Thank you for making it available to the community.

I have a good ole handlebar helper that I use on my projects for grouping:
```
   grouped_each: function (every, context, options) {
    var out = "", subcontext = [], i;
    if (context && context.length > 0) {
      for (i = 0; i < context.length; i++) {
        if (i > 0 && i % every === 0) {
          out += options.fn(subcontext);
          subcontext = [];
        }
        subcontext.push(context[i]);
      }
      out += options.fn(subcontext);
    }
    return out;         
  },
```
example:
```
{{#grouped_each 4 articles}}
	<div class="card-deck">
		{{#each this}}
    		<div class="card">
    			...
			</div>	
		{{/each}}
	</div>
{{/grouped_each}}
```

I am trying to switch to your library, but I am having a hard time finding a similar helper. Does it exist today? 

I noticed the #ifNth but it does not take into account the length of the array (only the modulo)

**Comment by jonschlinkert** [comment]
Thanks! Want to do a pr to add your helper? unit tests would be greatly appreciated too :)

**Comment by robsilva** [comment]
All set: https://github.com/helpers/handlebars-helpers/pull/249
Thanks again!

**Comment by jonschlinkert** [comment]
awesome, I'll get this merged on Monday, thanks!

**Comment by doowb** [comment]
Closed by #249
